main.go

```diff
@@ -19,21 +19,34 @@ package main
 import (
 	"log"
 
-	"github.com/spf13/cobra"
-
 	"github.com/joelanford/helm-operator/internal/cmd/run"
 	"github.com/joelanford/helm-operator/internal/cmd/version"
+	pluginv1 "github.com/joelanford/helm-operator/pkg/plugins/v1"
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/kubebuilder/pkg/cli"
 )
 
 func main() {
-	root := cobra.Command{
-		Use: "helm-operator",
+	commands := []*cobra.Command{
+		run.NewCmd(),
+		version.NewCmd(),
+	}
+	c, err := cli.New(
+		cli.WithCommandName("helm-operator"),
+		cli.WithPlugins(
+			&pluginv1.Plugin{},
```

Comment:
So `helm-operator` is going to have scaffolding subcommands? I'd prefer to split these CLI changes into a separate PR, and only add the plugin code in `pkg/plugins`.

Reply:
I can start a separate PR for the CLI changes.

What I understand from a previous discussion with Joe, is that this repo will be scaffolding the required files to build a `helm-operator` and once it's ready `operator-sdk` will import and run the vanilla helm-operator plugin from this repo, and separately apply its own phase 2 plugins (additional sub-commands), which are effectively overlaid on the base.